mac/setup/build.sh

```diff
@@ -71,25 +71,28 @@ cp textinputsource/textinputsource "$TARGETAPP/Contents/MacOS/textinputsource"
 
 # NOW TO notarize...
 xattr -rc "$TARGETAPP"
-execCodeSign direct --force --options runtime --deep --sign "${CERTIFICATE_ID}" "$TARGETAPP"
 
-#
-# Notarize the app (copied from ../build.sh)
-# TODO: merge notarization into function
-#
+if ! builder_is_ci_build || ! builder_is_ci_build_level_build; then
+  execCodeSign direct --force --options runtime --deep --sign "${CERTIFICATE_ID}" "$TARGETAPP"
 
-TARGET_ZIP_PATH="$TARGETPATH/Install Keyman.zip"
-TARGET_APP_PATH="$TARGETAPP"
+  #
+  # Notarize the app (copied from ../build.sh)
+  # TODO: merge notarization into function
+  #
 
-builder_heading "Zipping Install Keyman.app for notarization to $TARGET_ZIP_PATH"
+  TARGET_ZIP_PATH="$TARGETPATH/Install Keyman.zip"
+  TARGET_APP_PATH="$TARGETAPP"
 
-/usr/bin/ditto -c -k --keepParent "$TARGET_APP_PATH" "$TARGET_ZIP_PATH"
+  builder_heading "Zipping Install Keyman.app for notarization to $TARGET_ZIP_PATH"
 
-builder_heading "Uploading Install Keyman.zip to Apple for notarization"
-mac_notarize "$TARGETPATH" "$TARGET_ZIP_PATH"
+  /usr/bin/ditto -c -k --keepParent "$TARGET_APP_PATH" "$TARGET_ZIP_PATH"
 
-builder_heading "Attempting to staple notarization to Install Keyman.app"
-xcrun stapler staple "$TARGET_APP_PATH" || builder_die "stapler failed"
+  builder_heading "Uploading Install Keyman.zip to Apple for notarization"
+  mac_notarize "$TARGETPATH" "$TARGET_ZIP_PATH"
```

Comment:
OK... so we do notarize on this path, instead of the other, for local builds - meaning notarization _is_ covered for both control flows?